frontend/templates/troubleshooting/Resource.tsx

```diff
@@ -66,6 +70,32 @@ export function ProductResource({ product }: { product: SectionProduct }) {
    );
 }
 
+export function PartCollectionResource({
+   partCollection,
+}: {
+   partCollection: SectionPartCollection;
+}) {
+   const { title, url, description, imageUrl } = partCollection;
+
+   console.log('rendering part collection: ' + partCollection);
```

Comment:
```suggestion
```

I assume this should be removed?

Reply:
[db7d85b](https://github.com/iFixit/react-commerce/pull/2104/commits/db7d85b6220883af1d88545d1abf9818499b6930)